backend/typescript/models/testSession.model.ts

```diff
@@ -0,0 +1,81 @@
+import mongoose, { Schema, Document } from "mongoose";
+
+/**
+ * This interface holds information about the result of a single student
+ * on a test
+ */
+export interface Result {
+  /** the name of the student */
+  student: string;
+  /** the score of the student */
+  score: number;
+  /**
+   * a list corresponding to the question list with each fielding indicating
+   * whether the student got the question right or not
+   */
+  breakdown: [boolean];
```

Comment:
So, client has requested that we store student answers to questions (not just whether a question was correct or incorrect). I think by just changing this to `number` instead of `boolean` we should accomplish this? Let me know if this is an oversimplification.